schema/schema.go

```diff
@@ -0,0 +1,694 @@
+package schema
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+type Schema struct {
```

Comment:
Couldn't we split this up into sub structs for easier usability and readability?
Just a thought on first view.

Reply:
I think at first step it's a good idea to split each main property (like Core, Work) into a single struct.
Also, we can make something like ``StandardField`` struct for those fields that only have ``name`` and ``description`` inside them. We can also put an omitempty field inside this struct and use it for those structs that have ``format`` field too.

Reply:
Yeah that sounds like a good plan.